Umbraco-Cloud/Troubleshooting/Deployments/Deploy-Settings/index.md

```diff
@@ -19,6 +19,7 @@ The settings are set on the deploy element in the file. All settings are in seco
   <deploy sessionTimeout="1200" sourceDeployTimeout="1200" httpClientTimeout="1200"/>
 </settings>
 ```
+Do note that those settings have to be added on both the source and target environments to work.
```

Comment:
What do you think of adding this as a **note** instead? 🤔 

```suggestion
:::note
It's important that these settings are added to both the source and target environments in order to work.
:::
```

Reply:
Sure, the note is even better 😊